### PR TITLE
Feature/property service

### PR DESCRIPTION
**Pull request by blad90**
Implemented the **Property** service which has functionalities to add and retrieve properties based on the main RESTful operations.

### CODE REVIEW

File: property-service/pom.xml [hunk 1]
```diff
@@ -17,4 +17,39 @@
...
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-data-mongodb</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+<!--        <dependency>-->
```
Comment (jadder) [inline property-service/pom.xml:29]: why this dependency was comment and not actually included?

File: property-service/src/main/java/com/realtybuy/propertyservice/model/Property.java [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Data
+public class Property {
+    @Id
+    private String id;
+    private String name;
+    private String description;
+    private String address;
```
Comment (jadder) [inline property-service/src/main/java/com/realtybuy/propertyservice/model/Property.java:22]: for this member address we can use in string format, but in the next version we can implement a class with its own address atributes.

**Review by jadder — APPROVED** [review]
this PR looks good so far. minors posible changes could be implemented for next build.

File: property-service/pom.xml (hunk 1 above)
Comment (blad90) [reply property-service/pom.xml:29]: Because we'll include it on the next version when we add the Discovery Service.

**Pull request merged**